gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java

```diff
@@ -27,6 +27,7 @@
 import com.google.gwt.event.shared.GwtEvent;
 import com.google.gwt.event.shared.GwtEvent.Type;
 import com.google.gwt.event.shared.HasHandlers;
+import com.google.gwt.event.shared.UmbrellaException;
```

Comment:
Yeah, probably. I've put it to make the doc on that not red, I think. ;-)
Will remove.